source/horoscope/pages/landing.html

```diff
@@ -9,7 +9,21 @@
 </head>
 
 <body>
-    <h1 id="landing-title">Horoscope</h1>
+    <div class="header-container">
+        <h1 id="landing-title">Horoscope</h1>
+        <nav class="navbar">
+            <div class="menu-toggle" id="mobile-menu">
+                <span class="bar"></span>
```

Comment:
As opposed to creating the icon using span tags. It might make more sense to use an icon image. 

initialy thought it was unused code